package.json

```diff
@@ -90,9 +90,10 @@
     "@ckeditor/ckeditor5-ai": "41.3.1",
     "@ckeditor/ckeditor5-case-change": "41.3.1",
     "@ckeditor/ckeditor5-comments": "41.3.1",
+    "@ckeditor/ckeditor5-dev-build-tools": "^39.7.0-alpha.3",
     "@ckeditor/ckeditor5-dev-bump-year": "^39.0.0",
     "@ckeditor/ckeditor5-dev-ci": "^39.0.0",
-    "@ckeditor/ckeditor5-dev-dependency-checker": "^39.0.0",
+    "@ckeditor/ckeditor5-dev-dependency-checker": "^39.7.0-alpha.3",
```

Comment:
This package depends on `@ckeditor/ckeditor5-dev-utils`, which has been installed twice now. Let's consider releasing ckeditor5-dev as stable, too, even if it means adding minor changes to the `build-tools` package.